ipm_service/launch/ipm_service.launch

```diff
@@ -0,0 +1,7 @@
+<launch>
```

Comment:
Isn't this file kind of specific to bit-bots?

Reply:
No, it is a normal launch file without any bit-bots specific things.

Reply:
Why would someone want to use this launch file over running the node directly?
The only difference I see is using a "sim" launch argument instead of a "use_sim_time" parameter.

Reply:
I don't see it much easier, but the second one would be more natural because ``--ros-args -p use_sim_time:=true`` applies for any ROS2 node out there.

```sh
# Using launch
ros2 launch ipm_service ipm_service.launch sim:=true

# Using ros2 run
ros2 run ipm_service ipm_service --ros-args -p use_sim_time:=true
```

If you insist on having the launch file, shouldn't we at least have the parameter name ``use_sim_time`` and not ``sim``? - and at that point, is it still worth the launch file?

> Also, this is future prove as future changes might require launch file features and the "api" is still compatible

I'm not quite sure what you mean here, but I see launch files only useful when you have to start up multiple nodes, or you have more than one set of reasonable parameters for starting a node.

Reply:
Nonetheless, I don't see this as a major point stopping the progress in this PR, so we can keep it as is, and I'm going to move it to #32.